lib/internal/modules/cjs/loader.js

```diff
@@ -189,6 +197,25 @@ let requireDepth = 0;
 let isPreloading = false;
 let statCache = null;
 
+/**
+ * Internal method to be able to add recursion & tracing capabilities
```

Comment:
Why is recursion mentioned here? AFAICT it is unrelated?

Reply:
I removed the `requireDepth++`, so this comment is outdated, I will remove it.